src/Spryker/Configurator/DemoDateTimeConfiguratorPage/Theme/ConfiguratorApplication/src/services/configurator.service.ts

```diff
@@ -81,16 +93,18 @@ export class ConfiguratorService {
 
     setPrice(currency: string, displayData: DateConfiguration, priceMode: string, isAtStore: boolean): number {
         const isEur = currency.toLowerCase().trim() === 'eur';
-        const defaultPrice = isEur ? this.defaultPrices[0] : this.defaultPrices[1];
+        let defaultPrice = null;
         const isDayTimeEvening = displayData["Preferred time of the day"].toLowerCase().trim() === 'evening';
 
         if (!isAtStore || !displayData.Date || !isDayTimeEvening) {
             return defaultPrice;
         }
 
+        defaultPrice = isEur ? this.defaultPrices[0] : this.defaultPrices[1];
```

Comment:
```suggestion
        const isDayTimeEvening = displayData["Preferred time of the day"].toLowerCase().trim() === 'evening';

        if (!isAtStore || !displayData.Date || !isDayTimeEvening) {
            return null;
        }

        const defaultPrice = isEur ? this.defaultPrices[0] : this.defaultPrices[1];
```
Also, please add a new return type.